### PR TITLE
Lorawan refactor and improvements

### PR DESCRIPTION
**Pull request by ofauchon**
Lorawan refactor and improvements : 
    
- Remove default Lora modulation default initialisation in lorawan examples (Region Settings now takes care of this)
- Remove retries while receiving JoinAccept and increase LoraRX to 10s 
- Add constants for RegionSetting default frequencies
- Standardize lora modulation defaults in lora examples
- Add new AT+LW=NET,(ON|OFF) command in atcmd example

### CODE REVIEW

File: examples/lora/lorawan/atcmd/commands.go [hunk 1]
```diff
@@ -337,6 +337,19 @@ func delay(setting string) error {
 
 func lw(setting string) error {
 	cmd := "LW"
+
+	param, val, hasComma := strings.Cut(setting, ",")
+	if hasComma {
+		if param == "NET" {
+			if val == "ON" {
+				lorawan.SetPublicNetwork(true)
+				println("SET PUBLIC NET")
```
Comment (deadprogram) [inline examples/lora/lorawan/atcmd/commands.go]: Doesn't `writeCommandOutput(cmd, setting)` output enough already?

File: lora/lorawan/adaptor.go [hunk 2]
```diff
@@ -33,26 +33,38 @@ var (
...
 }
 
-func ApplyChannelConfig(ch *region.Channel) {
+// SetPublicNetwork defines Lora Sync Word according to network type (public/private)
+func SetPublicNetwork(enabled bool) {
+	ActiveRadio.SetPublicNetwork(enabled)
+}
+
+// ApplyChannelConfig sets current Lora modulation according to current regional settings
+func applyChannelConfig(ch *region.Channel) {
```
Comment (deadprogram) [inline lora/lorawan/adaptor.go:55]: :+1: on not exporting this.

File: examples/lora/lorawan/atcmd/commands.go (hunk 1 above)
Comment (ofauchon) [reply examples/lora/lorawan/atcmd/commands.go]: forgot this debug trace. Will fix

**Comment by deadprogram** [comment]
Closing since these changes were already picked up.

**Pull request closed without merging**